src/pheval/cli_pheval_utils.py

```diff
@@ -11,19 +16,120 @@
     metavar="FILE",
     help="Path to the semantic similarity profile to be scrambled.",
 )
-def scramble_semsim():
+def scramble_semsim(input: Path):
     """scramble_semsim"""
     print("running pheval_utils::scramble_semsim command")
 
 
-@click.command()
+@click.command("scramble-phenopackets")
 @click.option(
-    "--input",
-    "-i",
+    "--phenopacket-dir",
+    "-P",
+    metavar="PATH",
     required=True,
+    help="Path to phenopackets directory",
+)
+@click.option(
+    "--scramble-factor",
+    "-s",
+    metavar=float,
+    required=True,
+    default=0.5,
+    show_default=True,
+    help="Scramble factor for randomising phenopacket phenotypic profiles.",
+    type=float,
+)
+@click.option(
+    "--output-file-suffix",
+    "-o",
+    metavar="<str>",
+    required=True,
+    help="Suffix to append to output file",
+)
+@click.option(
+    "--output-dir",
+    "-O",
+    metavar="PATH",
+    required=True,
+    help="Path for creation of output directory",
+    default="noisy_phenopackets",
+)
+def scramble_phenopackets_command(
+    phenopacket_dir: Path,
+    scramble_factor: float,
+    output_file_suffix: str,
+    output_dir: Path,
+):
+    """Generate noisy phenopackets from existing ones."""
+    create_scrambled_phenopackets(output_dir, output_file_suffix, phenopacket_dir, scramble_factor)
+
+
+@click.command("semsim-comparison")
+@click.option(
+    "--semsim-left",
+    "-L",
+    required=True,
+    metavar="FILE",
+    help="Path to the first semantic similarity profile.",
+)
+@click.option(
+    "--semsim-right",
+    "-R",
+    required=True,
+    metavar="FILE",
+    help="Path to the second semantic similarity profile.",
+)
+@click.option(
+    "--score-column",
+    "-s",
+    required=True,
+    type=str,
+    help="Score column that will be used in comparison (e.g jaccard_similarity)",
+)
+@click.option(
+    "--analysis",
+    "-a",
+    default="heatmap",
+    type=str,
+    help="""There are two types of analysis:
+        heatmap - Generates a heatmap plot that shows the differences between the semantic similarity profiles using the
+        score column for this purpose. Defaults to "heatmap".
+        percentage_diff - Calculates the score column percentage difference between the semantic similarity profiles""",
+)
+@click.option(
+    "--output",
+    "-o",
     metavar="FILE",
-    help="Path to the phenopacket to be spiked.",
+    default="diff.semsim.tsv",
+    help="Output path for the difference tsv. Defaults to percentage_diff.semsim.tsv",
```

Comment:
The default file name for this option is different to the one stated in the `help` and then later on in the docstring